fdbsyncd/fdbsyncd.cpp

```diff
@@ -44,8 +46,29 @@ int main(int argc, char **argv)
              */
             if (sync.getRestartAssist()->isWarmStartInProgress())
             {
+                int pasttime = 0;                
                 sync.getRestartAssist()->readTablesToMap();
+                
+                while (!sync.isIntfRestoreDone())

```

Comment:
>isIntfRestoreDone [](start = 29, length = 17)

CPU is wasted on waiting. Could you subscribe Redis?

Reply:
As this is not a continuous busy wait ( sleep is present), this should not cause the cpu to be continuously busy. Also there is nothing for fdbsyncd to do until the interface info is populated to kernel after system warm-reboot, hence it needs to wait till such time.